File: Python/Majority-Element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        candidate=0
        times=0
        for i in nums:
            if(times==0):
                candidate=i
            if(i==candidate):
                times+=1
            else:
                times-=1
        return candidate

File: Java/Majority-Element.java
class Solution {
    public int majorityElement(int[] nums) {
        int c=0;
        int times=0;
        for(int num:nums){
            if(times==0){
                c=num;
            }
            if(num==c){
                times++;
            }
            else{
                times--;
            }
        }
        return c;
        }
        }
